spec/controllers/facility_accounts_controller_spec.rb

```diff
@@ -325,19 +325,37 @@
       is_expected.to render_template "show_statement_list"
     end
 
-    it "should show statement PDF" do
-      @params[:statement_id] = @statement.id
-      @params[:format] = "pdf"
-      maybe_grant_always_sign_in :director
-      do_request
-      expect(assigns(:account)).to eq(@account)
-      expect(assigns(:facility)).to eq(@authable)
-      expect(assigns(:statement)).to eq(@statement)
-      expect(response.content_type).to eq("application/pdf")
-      expect(response.body).to match(/%PDF-1.\d/)
-      is_expected.to render_template "statements/show"
-    end
+    describe "show" do
+      before do
+        @params[:statement_id] = @statement.id
+        @params[:format] = "pdf"
+      end
 
+      it "shows the statement PDF for a director" do
+        maybe_grant_always_sign_in :director
+        do_request
+        expect(assigns(:account)).to eq(@account)
+        expect(assigns(:facility)).to eq(@authable)
+        expect(assigns(:statement)).to eq(@statement)
+        expect(response.content_type).to eq("application/pdf")
+        expect(response.body).to match(/%PDF-1.\d/)
```

Comment:
Old issue, but it probably should be looking at the start of the string. `match(/\A%PDF-1\.\d+\b/)` maybe?